_posts/2020-07-16-1.0.0-release.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Apache Arrow 1.0.0 Release"
+date: "2020-07-16 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 1.0.0 release. This covers
+over 2 months of development work and includes [**XX resolved issues**][1]
+from [**XX distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## 1.0 Format Release
+
+The 1.0 major release indicates that the Arrow columnar format is declared
+stable, with [forward and backward compatibility guarantees][5].
+
+Integration testing, link to implementation matrix
+
+Format changes: unions, unsigned int dictionary indices, decimal bit width, feature enum.
+
+## Community
+
+Since the last release, we have added two new committers:
+
+* Liya Fan
+* Ji Liu
+
+Thank you for all your contributions!
+
+<!-- Acknowledge and link to any new committers and PMC members since the last release. See previous release announcements for examples. -->
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
+
+## Java notes
+
```

Comment:
The Java package introduces a number of low level changes in this release. Most notable are the work in support of allocating large arrow buffers and removing Netty from the public API. Users will have to update their dependencies to use one of the two supported allocators Netty: `arrow-memory-netty` or Unsafe (internal java api for direct memory) `arrow-memory-unsafe`.

The Java Vector implementation has improved its interoperability having verified `LargeVarChar`, `LargeBinary`, `LargeList`, `Union`, Extension types and duplicate field names in `Structs` are binary compatible with C++ and the specification.

For more on what’s in the 1.0.0 R package, see the [Java changelog][5].

Reply:
> For more on what’s in the 1.0.0 R package, see the [Java changelog][5].

If there is a Java changelog, please add a link to it below and give it a higher number here (5 is already taken). Also change where this says "R package".

If you do this as a "suggestion" comment, I can just merge it here.

Reply:
done!